test/unit/specs/components/staking/PageStaking.spec.js

```diff
@@ -1,26 +1,25 @@
-import setup from "../../../helpers/vuex-setup"
+import { shallowMount } from "@vue/test-utils"
 import PageStaking from "renderer/components/staking/PageStaking"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
 
-// TODO refactor according to new unit test standard
 describe(`PageStaking`, () => {
-  let wrapper, store
-  const { mount } = setup()
+  let wrapper, $store
+
+  const getters = {
+  }
 
   beforeEach(() => {
-    const instance = mount(PageStaking, {
-      doBefore: ({ store }) => {
-        store.commit(`setSignIn`, true)
-        store.commit(`setConnected`, true)
-        store.dispatch(`updateDelegates`)
+    $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      getters
```

Comment:
this component doesn't use `getters` anymore.

Reply:
Once this empty object is killed let's merge 🎉